store/index.js

```diff
@@ -46,5 +52,20 @@ export const getters = {
 
 export const actions = {
 
+    async getUnitList(store, params) {
+        return await axios.get(api_url + '/units', params).then(({data}) => {
+            return data;
+        }).catch((err) => {
+            console.log(err)
```

Comment:
Codacy has a fix for the issue: [Replace `console.log(err)` with `····console.log(err);`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9689045)
```suggestion
                console.log(err);
```